src/sentry/south_migrations/0324_auto__del_index_releaseenvironment_project_id__del_unique_releaseenvir.py

```diff
@@ -0,0 +1,876 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+from sentry.utils.db import is_postgres
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Removing unique constraint on 'ReleaseEnvironment', fields ['project_id', 'release_id', 'environment_id']
+        db.delete_unique('sentry_environmentrelease', ['project_id', 'release_id', 'environment_id'])
```

Comment:
Hmm, I'm going to have to research a tad, since this is different from what I normally do.